app/public/utils/url/urlUtils.js

```diff
@@ -28,3 +28,10 @@ export function urlSearchToParamsObject(search) {
     }
     return Object.fromEntries(search.slice(2).split('&').map((ent) => ent.split('=')));
 }
+
+export const formatParameter = (key, value) => `${key}=${value}`;
+
+export const href = (page, parameters) => {
+    const paramEntries = Object.entries(parameters);
+    return `?${[['page', page], ...paramEntries].map(([key, value]) => formatParameter(key, value)).join('&')}`;
+};
```

Comment:
- I'd rather you more meaningful name for function (and some docs as it is utility function)
- If it is utility function it's better to keep it as generic as possible, so parameter `page` can be easily merged into `parameters`

Reply:
- name changed to `buildHref`
- docs added
- page included in the parameters object